.github/workflows/build_test.yml

```diff
@@ -0,0 +1,124 @@
+# This is a basic workflow to help you get started with Actions
+
+name: build_test
+
+# Controls when the action will run.
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  push:
+    branches: [ master, simba-driverbuildtest-workflow ]
+  pull_request:
+    branches: [ master, simba-driverbuildtest-workflow ]
+
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build_test_windows:
+    strategy:
+      matrix:
+        php-version: ['7.4', '8.0', '8.1.21']
```

Comment:
why specific 8.1.21? can't we say 8.1?

Reply:
This is to solve the php version check issue here:
https://github.com/snowflakedb/pdo_snowflake/actions/runs/5718961615/job/15495872639
On windows for some reason we have to specify the minor version number to avoid such issue. I can't reproduce locally so likely it's related to PHP setup on github runner. It was set to 8.1.6 before this time I updated to the latest version of 8.1.21
https://github.com/snowflakedb/pdo_snowflake/blob/v1.2.7/.github/workflows/build_test_windows_php8-1_vs16.yml#L32